include/learning/InitializeProfile.h

```diff
@@ -0,0 +1,126 @@
+#ifndef InitializeProfile_H
```

Comment:
Can we change these variable names as well as the name of the files to be the same as the class name?